pkg/cli/environment.go

```diff
@@ -136,6 +148,18 @@ func envOr(name, def string) string {
 	return def
 }
 
+func envBoolOr(name string, def bool) bool {
```

Comment:
unittest for this func.

Reply:
resolved in c3fd382bea221ae755083235572eb282b6806067